### PR TITLE
TACC Environment Setup

### PR DESCRIPTION
**Pull request by jpvantassel**
# In this PR

- ✨ add scripts to build and start virtualenvironment on frontera and ls6.
- 🐛 move model to gpu to allow for gpu-based training.
- ✨ add slurm scripts for train, rollout, and render.

### CODE REVIEW

File: gns/train.py [hunk 1]
```diff
@@ -32,10 +32,10 @@
                     help='The path for saving outputs (e.g. rollouts).')
 flags.DEFINE_string('model_file', 'model.pt', help=('Model filename (.pt).'))
 
-flags.DEFINE_integer('ntraining_steps', int(2e7),
+flags.DEFINE_integer('ntraining_steps', int(5E5),
```
Comment (kks32) [inline gns/train.py:35]: set it back to 2e7 please

File: gns/train.py [hunk 2]
```diff
@@ -52,7 +52,7 @@
 KINEMATIC_PARTICLE_ID = 3
 
 device = torch.device('cuda' if torch.cuda.is_available() else 'cpu')
-
+#device = torch.device('cpu')
```
Comment (kks32) [inline gns/train.py:55]: remove this

File: gns/train.py [hunk 3]
```diff
@@ -250,7 +250,7 @@ def predict(
         metadata: json):
   """Predict rollouts.
 
-  Args:
```
Comment (kks32) [inline gns/train.py:253]: why is Args: removed?

**Review by kks32 — CHANGES_REQUESTED** [review]
remove the log files in slurm_script folder please

File: gns/train.py (hunk 3 above)
Comment (jpvantassel) [reply gns/train.py:253]: added back args

File: gns/train.py (hunk 2 above)
Comment (jpvantassel) [reply gns/train.py]: remove force cpu

File: gns/train.py (hunk 1 above)
Comment (jpvantassel) [reply gns/train.py]: switched back to 2e7

**Review by kks32 — APPROVED** [review]

**Pull request merged**